scripts/target.rb

```diff
@@ -12,6 +12,7 @@ class Target
       'libvirt'   => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2', nil, '66'),
       'gce'       => ImagefactoryMetadata.new('gce', nil, 'tar.gz', nil, '66'),
       'ec2'       => ImagefactoryMetadata.new('ec2', nil, 'vhd', 'zip', '66'),
+      'v2v'       => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2', nil, '10'),
```

Comment:
Can the target name be v2v_conf_host or something to differentiate that it's not a manageiq appliance?

Reply:
Changed to v2v_conv_host